doc/mp-build.md

```diff
@@ -48,9 +48,9 @@ If 'Other' is selected, the tool will prompt for a project path. If the path is
 
 #### Registry Credentials
 If the user is not logged into a container registry, `mp-build` will prompt
-the user to sign into one and enter their credentials. By default `mp-build` uses [harbor-repo.vmware.com](harbor-reop.vmware.com). Alternatively, the user can use 
+the user to sign into one and enter their credentials. By default `mp-build` uses the 'default_container_registry' specified in the global config.json file. Alternatively, the user can use
```

Comment:
If we have a global default, we need to clearly explain what it is, and how it is used, and why it is useful. Also make sure we aren't confusing the config files -- 'config.json' is local, but above we're referring to a 'global' config.json file, which doesn't exist.